docs/DeveloperGuide.md

```diff
@@ -1,29 +1,204 @@
+
+### Meeting Feature (PeiHao)
+The 'Meeting' Feature allows users to add a meeting, delete a meeting, find a meeting and print all the meetings.
+The 'Meeting' class in the entity package shows the attributes that a meeting object has: time and issue, both are of String type.
+An ArrayList of meetings is initialized in the MeetingManager. Three methods that implement the meetings are also inside.
+- addMeeting: Add a meeting to the meeting list.
+  ![](./uml/AddMeetingCommandSequenceDiagram-Add_Meeting_Sequence_Diagram.png)
+- deleteMeeting: Delete a meeting in the meeting list by its index.
+- printMeetings: Print all the meetings in the meeting list.
+
+When the user input a meeting command, the Parser will determine which command it is and return a command object.
+One of the meeting commands will be called from `AddMeetingCommand`, `DeleteMeetingCommand` and `ViewMeetingCommand`.
+Inside the command, the execute function will call the corresponding methods inside the MeetingManager and implement on the meeting list.
+
+### Deadline Feature (HuiQi)
+The Deadline Feature allows user to add Deadline objects in a deadline list.
+Deadline objects consists of a description String and a dueDate String.
+The User will also be able to delete deadlines, view deadline list.
+
+When the user input a deadline command, it will be parsed into Parser to determine which deadline command will be called.
+Then, the deadline command will be executed, and the respective method in the DeadlineManager class will be called, which are:
+#### addDeadline
+
+   Adds the deadline object to the ArrayList `deadlines`.
+
+   A deadline object can only be added to the deadlines list if it has a valid description and dueDate String.
+
+   If the addition was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been added.
+
+
+####deleteDeadline
+
+   Removes a deadline object from the ArrayList `deadlines` using the index given.
+   A deadline object can only be removed if the index is a valid index in the list.
+   If the deletion was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been deleted.
+
+3. printDeadlines
+   ![](./uml/ViewDeadlineDiagram.png)
+    Prints the ArrayList `deadlines` through TextUi class.
+
+### Staff Feature 
+The `Staff` Feature allows user to create, read, update, delete (CRUD) `Staff` objects in the list of staffs. 
+`Staff` list is managed by `StaffManager` Class similar to other entity objects. Here are hot it works for each functionality: 
+
+1. add_staff  
```

Comment:
Your type of bullet point (ie. numbered vs the black dots) is inconsistent throughout the entire DG. I recommend using a particular style for each type of content, ie. describing the functions vs the way a command is executed, or use the same type of bullet point throughout.